### PR TITLE
Away Missions: Now with 100% less RCDs

### LINKED ISSUE #822: RCDs can function in away missions. This is very bad.
RCDs should not be able to work in away missions, simply because it makes planetside content extremely dangerous to try, as anyone with an RCD can remove the floor and expose it to space, resulting in the entire atmosphere being drained and a lot of air simulation and movement. Totally immersion breaking to boot.

### PR DESCRIPTION
**Pull request by ike709**
Fixes #822 (If you can call it a fix.) Now RCDs won't work on any z-level above 6 (aka away missions). This is mainly to prevent immurshun ruining or something...

### CODE REVIEW

**Comment by ike709** [comment]
@WJohn @WJohn @WJohn

**Comment by WJohn** [comment]
Awesome, this is one of the biggest problems I was concerned about. Fires and bombs, while still capable of causing holes, are much less likely to be used.

:+1: x :100:

**Comment by ACCount12** [comment]
Just disable RCD floor deconstruction in z6+, so RCDs would be functional and safe.

**Comment by WJohn** [comment]
You know that works as well. As long as the floor cannot be destroyed and reveal space tiles in away missions I'm fine with it.

**Comment by ACCount12** [comment]
I really don't see a good reason to disable RCDs on away missions. This is another stupid nerf. Just disable floor deconstruction.

File: code/game/objects/items/weapons/RCD.dm [hunk 1]
```diff
@@ -149,7 +149,10 @@ RCD
 			return 0
 		if(!(istype(A, /turf) || istype(A, /obj/machinery/door/airlock)))
 			return 0
-
+		if(user.loc.z > 6)
```
Comment (GunHog) [inline code/game/objects/items/weapons/RCD.dm:152]: What happens if the loc is null?

Comment (ACCount12) [inline code/game/objects/items/weapons/RCD.dm:152]: Better to use get_turf(), but i see no way to use RCD while not being on turf.

**Comment by Atskadan** [comment]
thats silly

**Pull request closed without merging**